certs/api/transport.go

```diff
@@ -8,14 +8,14 @@ import (
 	"encoding/json"
 	"io"
 	"net/http"
-	"strconv"
-
-	"github.com/mainflux/mainflux/certs"
-	"github.com/mainflux/mainflux/pkg/errors"
 
 	kithttp "github.com/go-kit/kit/transport/http"
 	"github.com/go-zoo/bone"
 	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/certs"
+	intapihttp "github.com/mainflux/mainflux/internal/api/http"
```

Comment:
I like the prefix `internal` it's so obvious and I don't think it's too long (would rather sacrifice few more charts for readability). I would keep it everywhere and remove middle folder names like `api`. I think it's not needed `internalhttp` would be ok, even better `internalHTTP` or `internalErr` so it's super easy to spot.
Also regarding structure, I would remove `api` and keep `internal/http` only not sure why we have it in service at all... Anyway for an internal package, I would consider removing `api`.

Reply:
done!